data/seedData.py

```diff
@@ -140,6 +140,36 @@ def seedData():
     )
     user_xander_dander.save_to_db()
 
+    user_bobby_lowroll = UserModel(
+        email="broll@joinpdx.org",
+        firstName="Bobby",
+        lastName="Lowroll",
+        password="1234",
+        phone="555-555-5673",
+        archived=False,
+    )
+    user_bobby_lowroll.save_to_db()
+
+    user_cindy_norole = UserModel(
+        email="cindynorole@joinpdx.org",
+        firstName="Cindy",
+        lastName="Norole",
+        password="1234",
+        phone="555-555-5555",
+        archived=False,
+    )
+    user_cindy_norole.save_to_db()
+
+    user_joe_zero = UserModel(
+        email="joez@joinpdx.org",
+        firstName="Joe",
+        lastName="Zero",
+        password="1234",
+        phone="555-345-5555",
+        archived=False,
+    )
+    user_joe_zero.save_to_db()
+
```

Comment:
We don't really need more users in the seed file, we should probably just delete the `role=RoleEnum.Pending,` line from the existing ones that have that role. But feel free to leave your favorite one in there if you'd like 🙂 

Also, it looks like the reason there were no schema validation errors is because this file hasn't been fully switched over to use the schemas 😬.  Ideally, these should look more like the property entries (line 178+) except without variable names since the variables will no longer be used:
```python
# Pending users
    UserModel.create(
        payload={
            "email": "pending1@dwellingly.org",
            "firstName": "Anthony",
            "lastName": "Redding",
            "password": "1234",
            "phone": "555-555-5555",
        },
        schema=UserRegisterSchema,
    )
```

If you're up for it, I'd ask that all the "pending" users get updated to use this format.  I will open up an issue for refactoring the rest of the `seedData.py` entries to use schema validations as well.